packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.html

```diff
@@ -1,4 +1,54 @@
 <template>
-  <lightning-combobox name="sort" value={value} placeholder="Sort by" options={options} onchange={handleChange}>
-  </lightning-combobox>
+  <div class="slds-form-element">
+    <div class="slds-form-element__control">
+     
+      <div class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click" onmouseover={onMouseOver} onmouseout={onMouseOut} aria-expanded="true" aria-haspopup="listbox" role="combobox">
+        <div class="slds-combobox__form-element slds-input-has-icon slds-input-has-icon_left-right" role="none">
+          <span class="combobox__icon-container">
+            <lightning-icon class="slds-icon slds-input__icon slds-input__icon_left" icon-name="utility:filter" size="x-small" title="Filter icon"></lightning-icon>
```

Comment:
We shouldn't be using a filter icon for a sorting component. By doing that, we are breaking the SLDS UX conventions.

Reply:
I took it from the mock-ups. @adroletCoveo should I remove the icon entirely or replace it with another one?

Reply:
Good point! I used the icon to avoid having the _sort by_ label so that the combobox is aligned with the Results summary on the left. I agree that the icon is breaking the convention and we should remove it. But in this case we need the Sort by label. 

We can use the combobox as it is, but the alignement will be uneven. If possible, could the label be moved on the left of the combobox? That's how it's done in Atomic. It would also break the convention but with the label it would be clearer.

Reply:
Yes I can use the `lightning-combox` as is and use the built-in label. But then on top of the alignment issue, we would lose a little bit of the styling from the mockups (e.g. blue highlighting on hover).
The way it is now, the `Sort By` is there between the two icons, acting as a placeholder and is replaced once a selection is made. 

I can also just replace the filter icon with the icon offered for sorting components. Which would you prefer? @lbergeron  @nathanlb  what do you think?

![Screen Shot 2021-07-20 at 8 11 24 AM](https://user-images.githubusercontent.com/83351444/126323136-bd18cec5-3aad-4001-a8ed-9cb2d5b4858b.png)

Reply:
As discussed, let's see if we can use the `lightning-combobox` and move its label. If it doesn't work, I would replace the `filter` icon with the `sort` icon to remain coherent with SLDS.